src/main/resources/com/cloudbees/jenkins/plugins/docker_build_env/DockerBuildWrapper/help-protectedEnvironmentVariables.html

```diff
@@ -0,0 +1,4 @@
+<div>
+    A comma separated list defining what environment variables are protected.
+    Protected environment variables will not be overwritten by the docker plugin when making run or exec calls.
+</div>
```

Comment:
exposeDocker.html, dockerRegistryCredentials.html, dockerHost, and help.html do not have line breaks at the end either but I won't change those files in this PR